services/core-api/app/api/mines/permits/permit_extraction/create_permit_conditions.py

```diff
@@ -1,300 +1,124 @@
-import uuid
```

Comment:
This file got a bit unwieldy with the new changes, so I split it into a couple of classes, which is what most of the new code here is